pcm_osx.cpp

```diff
@@ -5,9 +5,10 @@
 
 class PcmOutputMacOSX: public PcmOutput {
 public:
-    PcmOutputMacOSX();
+    PcmOutputMacOSX(int rate);
     virtual ~PcmOutputMacOSX();
-    virtual int getSampleRate() { return 44100; }
+    virtual int getSampleRate() { return sample_rate; }
+    virtual void setSampleRate(int rate) { sample_rate = rate; }
```

Comment:
Why is there a `setSampleRate` when you can't actually change it after constructing the output?